### ISSUE TITLE
[MLH] Something went wrong: Sequence not found: WATER [MLH] SIDE GARDEN

### ISSUE THREAD

**Issue by AscendFB** [etcipnja/MLH#2]
Hey mate,

looks like your farmware broke with 2.2.1. When you insert anything else than "none" into `SEQUENCE NAME AFTER MOVE (EACH PLANT)` the famware crashes with the error message `Sequence not found: WATER [MLH] SIDE GARDEN`. Looks like those two lines may cause trouble.

```
265         if self.args['s']['after'].lower()!='none': self.args['s']['side']='Water [MLH] Side Garden'
266         else: self.args['s']['side']='None'
```

**Comment by etcipnja** [comment]
Yes! Thank you for the report. It looks like I need to keep my experiments in the separate branch. It is fixed in 2.2.2

BTW: do you know the way how to get SSH to the RaspPI? They seemed to lock it down completely... 
Otherwise I have deploy it via GitHub and it is pain in a back

**Comment by AscendFB** [comment]
That's not possible afaik. See this [FAQ entry.](https://github.com/FarmBot/farmbot_os/blob/staging/docs/FAQ.md#can-i-ssh-into-the-farmbot)
